Common/Data/SubscriptionManager.cs

```diff
@@ -206,7 +208,8 @@ public void AddConsolidator(Symbol symbol, IDataConsolidator consolidator)
                 {SecurityType.Option, new List<TickType>() { TickType.Quote, TickType.Trade, TickType.OpenInterest } },
                 {SecurityType.Cfd, new List<TickType>() { TickType.Quote } },
                 {SecurityType.Future, new List<TickType>() { TickType.Quote, TickType.Trade, TickType.OpenInterest } },
-                {SecurityType.Commodity, new List<TickType>() { TickType.Trade } }
+                {SecurityType.Commodity, new List<TickType>() { TickType.Trade } },
+                {SecurityType.Crypto, new List<TickType>() { TickType.Quote } }
```

Comment:
I think we should also add `TickType.Trade` here so we can have both quotes and trades. 
The Gdax PR #1090 will also require changes to emit both types of ticks.

Reply:
I see what you're getting at here. In the gdax brokerage I'm using the order stream to construct best bid/ask and emit a Quote tick, but you're saying Lean will do this for me if i emit TickType.Trade?

Reply:
The brokerage implementation should emit best bid/ask ticks as TickType.Quote and all trades ("match" message in GDAX) as TickType.Trade. 
Lean will receive these ticks and pass them to the algorithm as Tick objects (if Resolution.Tick) or consolidate quotes in QuoteBars and trades in TradeBars (for other resolutions).